src/utils/DateUtils.ts

```diff
@@ -1,24 +1,152 @@
 import { Options } from '../misc/Options';
 import { Utils } from './Utils';
 import { l } from '../strings/Strings';
-import { TimeSpan } from './TimeSpanUtils';
-import * as Globalize from 'globalize';
 import * as _ from 'underscore';
-
+import * as moment from 'moment';
+import * as d3 from 'd3';
+import days = d3.time.days;
+/**
+ * The `IDateToStringOptions` interface describes a set of options to use when converting a standard Date object to a
+ * string using the [ `dateToString` ]{@link DateUtils.dateToString}, [ `timeToString` ]{@link DateUtils.timeToString}, or the
```

Comment:
> [ ... ] using the [`dateToString`]{@link DateUtils.dateToString} or [`dateTimeToString`]{@link DateUtils.dateTimeToString} method [ ... ]

(sorry about that; since the `options` parameter of the `timeToString` method is pretty much deprecated, we should probably not mention it here after all :) )